src/api/index.ts

```diff
@@ -374,33 +376,33 @@ router.post(
         installation: {
           enabled: installation.enabled,
           id: installation.id,
-          jiraHost: installation.jiraHost,
-        },
+          jiraHost: installation.jiraHost
+        }
       });
 
     if (installation.enabled) {
-      respondWith('Installation already enabled');
+      respondWith("Installation already enabled");
       return;
     }
     await verifyInstallation(installation, req.log)();
     respondWith(
-      installation.enabled ? 'Verification successful' : 'Verification failed',
+      installation.enabled ? "Verification successful" : "Verification failed"
     );
-  },
+  }
 );
 
 router.post(
-  '/:installationId/migrate/:undo?',
+  "/:installationId/migrate/:undo?",
```

Comment:
This endpoint can be completely removed. It was done for DVCS to GitHub App migration testing and it doesn't exist in Jira anymore afaik. 

It is better to remove it to not confuse is during the current migration

Reply:
:+1: